### PR TITLE
Added Various Classifiers for all datasets. 

### PR DESCRIPTION
**Pull request by abbasidaniyal**
Monitored accuracy and F1 score. Plotted confusion matrix. Also used xgboost Classifier as it is very popular among Kagglers due to efficient performance.
In reference with #2

### CODE REVIEW

**Comment by mlopatka** [comment]
@abbasidaniyal Please let us know if you intend to continue with this PR.
I'm happy to discuss the review in our gitter channel if that will help you make progress.

**Comment by abbasidaniyal** [comment]
> @abbasidaniyal Please let us know if you intend to continue with this PR.
> I'm happy to discuss the review in our gitter channel if that will help you make progress.

I'm extremely sorry for the delay. I'll make the suggested changes and push ASAP. Will ask for guidance too. 
Thank you for waiting!

File: dev/abbasidaniyal/generated.ipynb [hunk 1]
```diff
@@ -0,0 +1,1807 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "temp =x.values #returns a numpy array\n",
+    "\n",
+    "min_max_scaler = preprocessing.MinMaxScaler()\n",
+    "x_scaled = min_max_scaler.fit_transform(temp)\n",
```
Comment (dzeber) [inline dev/abbasidaniyal/generated.ipynb]: Here you are scaling the full dataset prior to splitting train and test sets. This is discouraged as it means that some information from the test set (the scaling params) will get carried over to the training set. You should instead first do your train/test split, then fit the scaler on the training set, and finally transform both train and test. See [here](https://scikit-learn.org/stable/modules/preprocessing.html#scaling-features-to-a-range) for an example.

File: dev/abbasidaniyal/generated.ipynb [hunk 2]
```diff
@@ -0,0 +1,1807 @@
...
+    "\n",
+    "x_norm = pd.DataFrame(x_scaled)\n",
+    "x_train,x_test,y_train,y_test=train_test_split(x_norm,y,test_size=0.6)\n"
```
Comment (dzeber) [inline dev/abbasidaniyal/generated.ipynb]: You are using 40% training, 60% testing. This is valid, but I'm not sure if that's what you intended given your comment above?

**Comment by mlopatka** [comment]
This PR has been marked as inactive for over 7 days. Closing out. If work is resumed feel free to re-open and address outstanding feedback.
